backend/degree/models.py

```diff
@@ -549,6 +549,184 @@ class Meta:
         ]
 
 
+class Transcript(models.Model):
+    """
+    Not currently implemented
+    """
+
+    program = models.CharField(
+        max_length=255,
+        db_index=True,
+        help_text=dedent(
+            """
+            The user's current program (e.g. SEAS B.S.)
+            """
+        ),
+    )
+    courses = models.ForeignKey(
+        Course,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        help_text=dedent(
+            """
+            Courses already taken, prob a list of Course objects
+            """
+        ),
+    )
+
+
+class DegreeProfile(models.Model):
+    user_profile = models.OneToOneField(
+        UserProfile,
+        on_delete=models.CASCADE,
+        related_name="degree_profile",
+        help_text="extending the user profile class",
+    )
+
+    transcript = models.OneToOneField(
+        Transcript,
+        on_delete=models.CASCADE,
+        null=True,
+        related_name="degree_profile",
+        help_text="The user's uploaded transcript parsed into a Transcript object (optional)",
+    )
+
+    graduation_date = models.CharField(
+        max_length=5,
+        help_text=dedent(
+            """
+            The user's expected graduation date (of the form YYYYx where x is A [for spring],
+            B [summer], or C [fall]), e.g. `2019C` for fall 2019)
+            """
+        ),
+    )
+
+    degrees = models.ManyToManyField(
```

Comment:
Why do we store degrees here? How would we display this on the onboarding panel.